package.json

```diff
@@ -12,7 +12,7 @@
     "mongoose": "^5.11.16"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "nodemon"
```

Comment:
Hmm. Add "prod": "node index.js" 

I think heroku use npm start by default. We need to remember to change it later or to verify what I am saying